Build/Projects/MonoGame.Framework.definition

```diff
@@ -1190,6 +1190,9 @@
     <Compile Include="Utilities\ReflectionHelpers.Legacy.cs" >
       <Platforms>Angle,Linux,MacOS,Windows,WindowsGL,Web</Platforms>
     </Compile>
+    <Compile Include="Utilities\TimerHelper.cs">
+      <Platforms>Windows,WindowsGL</Platforms>
```

Comment:
No biggy, but ```WindowsGL``` should be removed here (it's not an issue that ```TimerHelper``` is included, as long as it's not used in DesktopGL, because ```ntdll.dll``` won't exist on Mac and Linux).

Reply:
Yeah, @Jjagg raised this also in his review. But he said it's fine as is for now, and he'd make some changes later.

tbh, I'm a little confused with this "DesktopGL" build (I only use the DirectX build). So to clarify - Is the "WindowsGL" project definition actually the "DesktopGL" build, which is also used on Mac and Linux? So if that's the case, then what are the "Linux" and "MacOS" project definitions used for?

Reply:
Protobuild needs a definition for each platform. Even though the DesktopGL builds for Windows and Linux where the same (or rather should be) they both need definitions. But the way we structure the .definition files is pretty bad because it uses platforms instead of services (e.g. DesktopGL).

Reply:
Thanks @Jjagg, I think I understand now. So ultimately, there will be one build for "DesktopGL" which will work on all 3 platforms. But right now, there are still actually 3 separate builds. And this is why "WindowsGL" running this code is not a problem right now - because it's currently only used for the Windows build. But it is an issue in working towards having one build for all 3 platforms. Have I got this right?

Reply:
Yes, kind of. I'm not sure what the plan for Mac is, but AFAIK before this PR the Windows and Linux builds were identical and now a Windows build won't run on Linux or Mac.

Reply:
Ok. Well I understand what needs to be done to fix this up now, so just say the word if you'd like me to submit a PR.

Reply:
Yeah, DesktopGL = WindowsGL = Linux. They used to be very different but they are now all fused and are the same.
Regarding Mac, both the "GL" target and the "Mac" target are almost identical, except that the "Mac" one has a platform specific video player implementation like Jjagg mentioned.

(Confusing, yes. I guess that at some point we should work toward having a unique video player for GL and fuse all those platforms.)

And sorry for not reading thoroughly Jjagg's review!

Reply:
Yup, disable this code for WindowsGL, that is submit a PR for it.

Reply:
Thanks for the clarification guys! I'll get a PR submitted shortly to fix this up.